### DISCUSSION TITLE
Custom/fluid width columns per board

### DISCUSSION THREAD

**Discussion by obbardc** [JordanKnott/taskcafe#68] [Ideas]
I have a 1080p screen and when showing a board only ~7 columns fit on the screen without scrolling. I'd like to fit 8 on the screen without scrolling ;-).

Currently by default the columns are `272px` wide; it would be nice to be able to either modify this per board OR have an option per board to make the columns "fluid" so that all columns would fit inside the window, defaulting to a minimum and maximum width.

**Comment by JordanKnott** [comment]
Thanks for the suggestion! 

The ability to specify column width is interesting, perhaps I can this feature in the future! I do plan on changing the default width but I had planned on making them a bit bigger (280px?).

As far as the fluid columns, I envision a lot of issues trying to get that to work well (e.g when you start getting a lot of lists) - that likely won't be some ill support 😞

**Comment by MartynKeigher** [comment]
I like the idea of setting a width on the 'columns' too. I use a 4K monitor here so having the ability to INCREASE the width would be something that I would happily do to get more info (for the notes of each task) on my screen.

**Comment by BeGj** [comment]
Would be nice if the boards could flex to 100% width total.
